src/fsipy/automatedPreprocessing/preprocessing_common.py

```diff
@@ -203,3 +204,47 @@ def convert_xml_mesh_to_hdf5(file_name_xml_mesh: str, scaling_factor: float = 0.
     hdf.write(mesh, "/mesh")
     hdf.write(boundaries, "/boundaries")
     hdf.write(domains, "/domains")
+
+
+def convert_vtu_mesh_to_xdmf(file_name_vtu_mesh: str, file_name_xdmf_mesh: str) -> None:
```

Comment:
Should we add a note that this is only meant to be executed in serial?
As `meshio` has no concept of MPI it would read the mesh on all processes, which could cause some funky race conditions in parallel.